src/app/screens/ConfirmAddAccount/index.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { act, render, screen } from "@testing-library/react";
+import { MemoryRouter } from "react-router-dom";
+import type { OriginData } from "~/types";
+
+import ConfirmAddAccount from "./index";
+
+const mockOrigin: OriginData = {
+  location: "https://getalby.com/demo",
+  domain: "https://getalby.com",
+  host: "getalby.com",
+  pathname: "/demo",
+  name: "Alby",
+  description: "",
+  icon: "https://getalby.com/assets/alby-503261fa1b83c396b7ba8d927db7072d15fea5a84d387a654c5d0a2cefd44604.svg",
+  metaData: {
+    title: "Alby Demo",
+    url: "https://getalby.com/demo",
+    provider: "Alby",
+    image:
+      "https://getalby.com/assets/alby-503261fa1b83c396b7ba8d927db7072d15fea5a84d387a654c5d0a2cefd44604.svg",
+    icon: "https://getalby.com/favicon.ico",
+  },
+  external: true,
+};
+
+jest.mock("~/app/hooks/useNavigationState", () => {
+  return {
+    useNavigationState: jest.fn(() => ({
+      origin: mockOrigin,
+      args: {
+        message: "Test message",
+      },
+    })),
+  };
+});
+
+describe("ConfirmAddAccount", () => {
+  test("render", async () => {
+    await act(async () => {
+      render(
+        <MemoryRouter>
+          <ConfirmAddAccount />
+        </MemoryRouter>
+      );
+    });
+
+    expect(
+      await screen.findByText(
+        "This website wants to add a wallet of the following type to Alby:"
+      )
+    ).toBeInTheDocument();
+    expect(await screen.findByText("Test message")).toBeInTheDocument();
```

Comment:
[This fails in CI](https://github.com/getAlby/lightning-browser-extension/actions/runs/3766670301/jobs/6403420007), did this work locally?

Reply:
I haven't ran the tests yet, I'm still working on this PR (I'll make some adjustments tomorrow)